healthcareai/develop_supervised_model.py

```diff
@@ -303,7 +304,8 @@ def write_classification_metrics_to_json(self):
         with open('classification_metrics.json', 'w') as fp:
             json.dump(output, fp, indent=4, sort_keys=True)
 
-
+    def write_model_to_pickle(self):
+        pickle.dump( self.results['best_model'], open( "classification_best_model.pkl", "wb" ) )
```

Comment:
We may want to allow the user to specify the filename (and maybe include a timestamp in the filename).

Also, I think (but am not certain) that you'll want to close the file after dumping it out using either .close() or the context manager like

```
with open("filename.pkl", "wb") as output_file:
    pickle.dump(model, output_file)
```